### ISSUE TITLE
Tooltips placement wrong

### ISSUE THREAD

**Issue by Ahmadre** [flutter/flutter#52571]
## Steps to Reproduce

**Related Code Snipped:** 

```dart
Scaffold(
              key: _scaffoldKey,
              appBar: AppBar(
                elevation: 0,
                backgroundColor: Colors.transparent,
                centerTitle: true,
                actions: <Widget>[
                  IconButton(icon: Icon(Icons.view_carousel), tooltip: 'Vorschau', onPressed: () => viewImageGallery())
])
);
```

<img width="338" alt="Bildschirmfoto 2020-03-13 um 22 22 39" src="https://user-images.githubusercontent.com/18512224/76660493-55041300-6579-11ea-8bb5-22dae1247772.png">

<img width="335" alt="Bildschirmfoto 2020-03-13 um 22 22 45" src="https://user-images.githubusercontent.com/18512224/76660503-59c8c700-6579-11ea-9084-e45d70b026ee.png">

**Expected results:** 
Tooltips are placed correctly

**Actual results:**
Tooltips on Flutter Web aren't placed correctly

<details>
  <summary>Logs</summary>

### Verbose
```
[  +35 ms] executing:
[/Users/rebar/Documents/flutter/] git -c
log.showSignature=false log -n 1 --pretty=format:%H
[  +39 ms] Exit code 0 from: git -c
log.showSignature=false log -n 1 --pretty=format:%H
[        ] 7cbe55cf1abedb97e3e59da9a203dc20e1b9d25a
[        ] executing:
[/Users/rebar/Documents/flutter/] git describe --match
v*.*.* --first-parent --long --tags
[  +32 ms] Exit code 0 from: git describe --match
v*.*.* --first-parent --long --tags
[        ] v1.15.3-165-g7cbe55cf1
[   +9 ms] executing:
[/Users/rebar/Documents/flutter/] git rev-parse
--abbrev-ref --symbolic @{u}
[   +6 ms] Exit code 0 from: git rev-parse
--abbrev-ref --symbolic @{u}
[        ] origin/master
[        ] executing:
[/Users/rebar/Documents/flutter/] git ls-remote
--get-url origin
[   +6 ms] Exit code 0 from: git ls-remote --get-url
origin
[   +1 ms] https://github.com/flutter/flutter.git
[  +61 ms] executing:
[/Users/rebar/Documents/flutter/] git rev-parse
--abbrev-ref HEAD
[   +8 ms] Exit code 0 from: git rev-parse
--abbrev-ref HEAD
[        ] master
[   +5 ms] executing: sw_vers -productName
[  +17 ms] Exit code 0 from: sw_vers -productName
[        ] Mac OS X
[        ] executing: sw_vers -productVersion
[  +18 ms] Exit code 0 from: sw_vers -productVersion
[        ] 10.15.3
[        ] executing: sw_vers -buildVersion
[  +17 ms] Exit code 0 from: sw_vers -buildVersion
[        ] 19D76
[  +29 ms] Artifact Instance of
'AndroidMavenArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'AndroidGenSnapshotArtifacts' is not required,
skipping update.
[        ] Artifact Instance of
'AndroidInternalBuildArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts'
is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not
required, skipping update.
[   +3 ms] Artifact Instance of
'WindowsEngineArtifacts' is not required, skipping
update.
[        ] Artifact Instance of 'MacOSEngineArtifacts'
is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts'
is not required, skipping update.
[        ] Artifact Instance of
'LinuxFuchsiaSDKArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'MacOSFuchsiaSDKArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'FlutterRunnerSDKArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'FlutterRunnerDebugSymbols' is not required, skipping
update.
[  +26 ms] executing:
/Users/rebar/Library/Android/sdk/platform-tools/adb
devices -l
[  +10 ms] Exit code 0 from:
/Users/rebar/Library/Android/sdk/platform-tools/adb
devices -l
[   +1 ms] List of devices attached
[  +10 ms] executing: /usr/bin/xcode-select
--print-path
[   +7 ms] Exit code 0 from: /usr/bin/xcode-select
--print-path
[   +1 ms] /Applications/Xcode.app/Contents/Developer
[   +1 ms] executing: /usr/bin/xcodebuild -version
[ +102 ms] Exit code 0 from: /usr/bin/xcodebuild
-version
[   +1 ms] Xcode 11.3.1
           Build version 11C504
[   +2 ms] executing: xcrun --find xcdevice
[  +10 ms] Exit code 0 from: xcrun --find xcdevice
[        ]
/Applications/Xcode.app/Contents/Developer/usr/bin/xcd
evice
[   +2 ms] executing: xcrun xcdevice list --timeout 1
[   +6 ms] /usr/bin/xcrun simctl list --json devices
[+1694 ms] [
                      {
                        "simulator" : true,
                        "operatingSystemVersion" :
                        "13.3 (17C45)",
                        "available" : true,
                        "platform" :
                        "com.apple.platform.iphonesimu
                        lator",
                        "modelCode" : "iPhone10,5",
                        "identifier" :
                        "830E878F-F0C5-4FFF-B0F4-62E25
                        F9955E8",
                        "architecture" : "x86_64",
                        "modelName" : "iPhone 8 Plus",
                        "name" : "iPhone 8 Plus"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" :
                        "13.3 (17C45)",
                        "available" : true,
                        "platform" :
                        "com.apple.platform.iphonesimu
                        lator",
                        "modelCode" : "iPad8,5",
                        "identifier" :
                        "3678658B-C6F8-4EA5-B7F4-0EEE9
                        326BBCE",
                        "architecture" : "x86_64",
                        "modelName" : "iPad Pro
                        (12.9-inch) (3rd generation)",
                        "name" : "iPad Pro (12.9-inch)
                        (3rd generation)"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" :
                        "13.3 (17C45)",
                        "available" : true,
                        "platform" :
                        "com.apple.platform.iphonesimu
                        lator",
                        "modelCode" : "iPhone12,5",
                        "identifier" :
                        "4C3E4855-3033-4F8B-BD55-ED013
                        B547D7B",
                        "architecture" : "x86_64",
                        "modelName" : "iPhone 11 Pro
                        Max",
                        "name" : "iPhone 11 Pro Max"
                      }
                    ]
[   +9 ms] Artifact Instance of
'AndroidMavenArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'AndroidGenSnapshotArtifacts' is not required,
skipping update.
[        ] Artifact Instance of
'AndroidInternalBuildArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts'
is not required, skipping update.
[   +6 ms] Artifact Instance of
'WindowsEngineArtifacts' is not required, skipping
update.
[        ] Artifact Instance of 'MacOSEngineArtifacts'
is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts'
is not required, skipping update.
[        ] Artifact Instance of
'LinuxFuchsiaSDKArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'MacOSFuchsiaSDKArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'FlutterRunnerSDKArtifacts' is not required, skipping
update.
[        ] Artifact Instance of
'FlutterRunnerDebugSymbols' is not required, skipping
update.
[ +167 ms] Found plugin firebase_auth at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth-0.15.4/
[  +14 ms] Found plugin firebase_auth_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth_web-0.1.1+4/
[   +3 ms] Found plugin firebase_core at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core-0.4.3+2/
[   +5 ms] Found plugin firebase_core_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core_web-0.1.1+2/
[ +131 ms] Found plugin url_launcher at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher-5.4.2/
[   +2 ms] Found plugin url_launcher_macos at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_macos-0.0.1+4/
[   +3 ms] Found plugin url_launcher_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_web-0.1.1+1/
[   +4 ms] Found plugin video_player at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player-0.10.8+1/
[   +3 ms] Found plugin video_player_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player_web-0.1.2+1/
[  +60 ms] Found plugin firebase_auth at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth-0.15.4/
[   +1 ms] Found plugin firebase_auth_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth_web-0.1.1+4/
[   +1 ms] Found plugin firebase_core at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core-0.4.3+2/
[   +1 ms] Found plugin firebase_core_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core_web-0.1.1+2/
[  +43 ms] Found plugin url_launcher at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher-5.4.2/
[        ] Found plugin url_launcher_macos at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_macos-0.0.1+4/
[   +1 ms] Found plugin url_launcher_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_web-0.1.1+1/
[   +1 ms] Found plugin video_player at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player-0.10.8+1/
[   +1 ms] Found plugin video_player_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player_web-0.1.2+1/
[  +66 ms] Generating
/Users/rebar/Documents/telexiom/LUITS_App/LUITS_Web/lu
itsapp/android/app/src/main/java/io/flutter/plugins/Ge
neratedPluginRegistrant.java
[ +131 ms] Launching lib/main.dart on Chrome in debug
mode...
[  +90 ms] Serving DevTools at http://127.0.0.1:49519

[   +9 ms] Updating assets
[ +388 ms] Syncing files to device Chrome...
[  +24 ms] Found plugin firebase_auth at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth-0.15.4/
[   +2 ms] Found plugin firebase_auth_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth_web-0.1.1+4/
[   +1 ms] Found plugin firebase_core at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core-0.4.3+2/
[   +2 ms] Found plugin firebase_core_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core_web-0.1.1+2/
[  +44 ms] Found plugin url_launcher at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher-5.4.2/
[   +1 ms] Found plugin url_launcher_macos at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_macos-0.0.1+4/
[   +2 ms] Found plugin url_launcher_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_web-0.1.1+1/
[   +2 ms] Found plugin video_player at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player-0.10.8+1/
[   +2 ms] Found plugin video_player_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player_web-0.1.2+1/
[  +23 ms] Found plugin firebase_auth at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth-0.15.4/
[   +1 ms] Found plugin firebase_auth_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_auth_web-0.1.1+4/
[   +2 ms] Found plugin firebase_core at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core-0.4.3+2/
[   +2 ms] Found plugin firebase_core_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/fireba
se_core_web-0.1.1+2/
[  +44 ms] Found plugin url_launcher at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher-5.4.2/
[   +1 ms] Found plugin url_launcher_macos at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_macos-0.0.1+4/
[   +1 ms] Found plugin url_launcher_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/url_la
uncher_web-0.1.1+1/
[   +1 ms] Found plugin video_player at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player-0.10.8+1/
[   +1 ms] Found plugin video_player_web at
/Users/rebar/.pub-cache/hosted/pub.dartlang.org/video_
player_web-0.1.2+1/
[  +10 ms] Generating
/Users/rebar/Documents/telexiom/LUITS_App/LUITS_Web/lu
itsapp/android/app/src/main/java/io/flutter/plugins/Ge
neratedPluginRegistrant.java
[ +121 ms] <- reset
[  +30 ms]
/Users/rebar/Documents/flutter/bin/cache/dart-sdk/bin/
dart
/Users/rebar/Documents/flutter/bin/cache/artifacts/eng
ine/darwin-x64/frontend_server.dart.snapshot
--sdk-root
/Users/rebar/Documents/flutter/bin/cache/flutter_web_s
dk/ --incremental --target=dartdevc
-Ddart.developer.causal_async_stacks=true
--output-dill
/var/folders/j9/qv2plg1n6qn_5kgl40wb1mc80000gn/T/flutt
er_tool.uxuLBF/app.dill --packages
/Users/rebar/Documents/telexiom/LUITS_App/LUITS_Web/lu
itsapp/.packages -Ddart.vm.profile=false
-Ddart.vm.product=false
--bytecode-options=source-positions,local-var-info,deb
ugger-stops,instance-field-initializers,keep-unreachab
le-code,avoid-closure-call-instructions
--enable-asserts --track-widget-creation
--filesystem-root
/var/folders/j9/qv2plg1n6qn_5kgl40wb1mc80000gn/T/flutt
er_tools.wFxB5S --filesystem-scheme org-dartlang-app
--platform
file:///Users/rebar/Documents/flutter/bin/cache/flutte
r_web_sdk/kernel/flutter_ddc_sdk.dill
[  +13 ms] <- compile
org-dartlang-app:///web_entrypoint.dart
[+17403 ms] Syncing files to device Chrome...
(completed in 17.732ms, longer than expected)
[   +1 ms] Synced 25.5MB.
[        ] <- accept
[ +246 ms] [CHROME]:
[   +1 ms] [CHROME]:DevTools listening on
ws://127.0.0.1:49582/devtools/browser/b29d1b7c-32ec-4c
b1-9216-482c2655bb27
[+2916 ms] Debug service listening on
ws://127.0.0.1:49607/5qF3iT7tU4g=

[ +395 ms] Debug service listening on
                    ws://127.0.0.1:49607/5qF3iT7tU4g=
[   +2 ms] Warning: Flutter's support for web
development is not
           stable yet and hasn't
[        ] been thoroughly tested in production
environments.
[        ] For more information see
https://flutter.dev/web
[   +1 ms] 🔥  To hot restart changes while running,
press "r".
           To hot restart (and refresh the browser),
press "R".
[   +2 ms] For a more detailed help message, press
"h". To quit,
           press "q".
```

### Analyze
```
Analyzing luitsapp...                                                   

   info • The value of the field '_auth' isn't used • lib/Screens/CredentialScreens/login.dart:23:22 • unused_field
   info • This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its
          instance fields aren't final: MedicalTile.documentID, MedicalTile.order, MedicalTile.documentTitle •
          lib/Screens/Dashboard/Components/MedicalTile.dart:3:7 • must_be_immutable
   info • This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its
          instance fields aren't final: Editor.titleText, Editor.document, Editor.iconData •
          lib/Screens/Dashboard/Editor.dart:11:7 • must_be_immutable
   info • Unused import: 'package:luitsapp/Screens/Dashboard/SOPView/SOPView.dart' •
          lib/Screens/Dashboard/SubCategory/SubCategory.dart:6:8 • unused_import
   info • This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its
          instance fields aren't final: SubCategory.referencedCategory, SubCategory.subList •
          lib/Screens/Dashboard/SubCategory/SubCategory.dart:21:7 • must_be_immutable
   info • This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its
          instance fields aren't final: Dashboard.userData • lib/Screens/Dashboard/dashboard.dart:21:7 •
          must_be_immutable
   info • The value of the field '_videosController' isn't used • lib/Screens/Profile/Profile.dart:11:18 • unused_field
   info • This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its
          instance fields aren't final: Settings.user • lib/Screens/Settings/index.dart:14:7 • must_be_immutable
   info • This function has a return type of 'Future<OAuthCredential>', but doesn't end with a return statement •
          lib/Screens/Settings/index.dart:51:30 • missing_return
   info • This function has a return type of 'Future<OAuthCredential>', but doesn't end with a return statement •
          lib/Screens/Settings/index.dart:64:30 • missing_return
   info • The declaration '_scalingMargin' isn't referenced •
          lib/utils/ResponsiveComponents/responsive_widgets.dart:8:8 • unused_element
   info • Don't explicitly initialize variables to null • lib/utils/ResponsiveComponents/responsive_widgets.dart:11:8 •
          avoid_init_to_null
   info • Don't explicitly initialize variables to null • lib/utils/ResponsiveComponents/responsive_widgets.dart:12:8 •
          avoid_init_to_null

13 issues found. (ran in 2.4s)
```

### Doctor

```
[✓] Flutter (Channel master, v1.15.4-pre.165, on Mac OS X 10.15.3 19D76, locale de-DE)
    • Flutter version 1.15.4-pre.165 at /Users/rebar/Documents/flutter
    • Framework revision 7cbe55cf1a (33 hours ago), 2020-02-25 14:56:04 -0800
    • Engine revision 888a62cf22
    • Dart version 2.8.0 (build 2.8.0-dev.10.0 73f6d15665)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/rebar/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.8.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 41.1.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.42.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.8.1

[✓] Connected device (2 available)
    • Chrome     • chrome     • web-javascript • Google Chrome 80.0.3987.122
    • Web Server • web-server • web-javascript • Flutter Tools

• No issues found!
```

</details>

**Comment by Ahmadre** [comment]
To all flutter devs: I don't expect any answer for now, because also here in germany we're all in Home-Office and a lot places are closed. It's totally ok, if my issues stay still until this pandemy is over. Hope you all are okay and the siuation will get better :) Greetings to all of you

**Comment by yjbanov** [comment]
This looks like a general framework issue, not web-specific, but please feel free to assign to platform-web if this is web-specific. /cc @goderbauer

**Comment by goderbauer** [comment]
Can you post a minimal, but complete code sample that reproduces the issue? If I throw your code snippet into a simple app, the tooltips show up at the correct location as expected: https://dartpad.dartlang.org/d88c8a88741bc51fc3150d077f4c6d5f

**Comment by Ahmadre** [comment]
@goderbauer I tried to reproduce it as well with a minimal code: https://dartpad.dartlang.org/51714e17cf75df0f7c2dedf4078a150a

I inserted Navigation_rail, my approach for bigger screens with Row and Expanded, but somehow I can't get the drawer bigger like my screenshot above. I don't know how to simulate a big screen (minimum 4:3 or 16:9) on DartPad. I can't make the show area bigger...

**Comment by Ahmadre** [comment]
Here's a GIF showing that weird error on Flutter Debug App. The tooltip is not even showing when the browser window is small, but when I enlarge it, that happens:

![ezgif com-video-to-gif (3)](https://user-images.githubusercontent.com/18512224/77132383-83915a80-6a5f-11ea-9640-99d6b0c3d33d.gif)

It's the same code as mentioned in my dartpad above (besides the List.generate).

**Comment by TahaTesser** [comment]
Hi @Ahmadre 
Not able to reproduce this issue using your code sample https://github.com/flutter/flutter/issues/52571#issuecomment-601502818

<img width="234" alt="Screenshot 2020-07-08 at 5 25 49 PM" src="https://user-images.githubusercontent.com/48603081/86915798-35a07f80-c140-11ea-90ba-8b8f840270bc.png">


Are you still experiencing this issue on latest `dev`? if yes, please steps to reproduce with your code sample

<details> 
<summary>flutter doctor -v</summary> 

```
[✓] Flutter (Channel dev, 1.20.0-3.0.pre, on Mac OS X 10.15.5 19F101, locale en-GB)
    • Flutter version 1.20.0-3.0.pre at /Users/taha/Code/flutter_dev
    • Framework revision 0af027f805 (4 days ago), 2020-07-04 12:19:20 -0700
    • Engine revision a751393804
    • Dart version 2.9.0 (build 2.9.0-20.0.dev 22da8934ac)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at /Users/taha/Code/sdk
    • Platform android-30, build-tools 30.0.0
    • ANDROID_HOME = /Users/taha/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.46.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.1

 
[✓] Connected device (3 available)            
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS 13.5.1
    • Web Server (web)       • web-server                • web-javascript • Flutter Tools
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome
      83.0.4103.116

• No issues found!

``` 

</details>

**Comment by romathebest** [comment]
I have the same issue on the web. It happens if I put a scaffold in other shifted container. The same problem with mouse hover effect when using AutoCompleteTextField or TypeAheadFormField
![Annotation 2020-07-15 122524](https://user-images.githubusercontent.com/3680216/87529605-e904f900-c697-11ea-8588-7ad8b623b3e9.png)

**Comment by TahaTesser** [comment]
@romathebest 
Can you please provide your `flutter doctor -v` and a complete reproducible minimal code sample
Thank you

**Comment by romathebest** [comment]
@TahaTesser after an hour of trying to reproduce it for example I finally got where is the problem. It is in navigation and page routing. I have created TestLayout with header, footer, and make navigation in the center. In such a case, all tooltips are shifted. 
![Annotation 2020-07-16 170241](https://user-images.githubusercontent.com/3680216/87680519-33fd3a00-c786-11ea-99a5-7bec6c347079.png)


Here is a full main.dart file of the test app:

```
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

Route<dynamic> generateRoute(RouteSettings settings) {
  return MaterialPageRoute<dynamic>(builder: (context) => MyHomePage());
}

class TestLayout extends StatelessWidget {
  final Widget child;
  
  const TestLayout({Key key, this.child}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
        backgroundColor: Colors.white70,
        body: SafeArea(
            child: Column(
              crossAxisAlignment: CrossAxisAlignment.stretch,
              children: [
                Container(
                  height: 100,
                    alignment: Alignment.topCenter,
                    child: ConstrainedBox(
                      constraints: BoxConstraints(maxWidth: 1200),
                      child: Center(child: Text('Header')),
                    ),
                ),
                Expanded(
                    child: Container(
                      padding: const EdgeInsets.all(70),
                      alignment: Alignment.topCenter,
                      child: ConstrainedBox(
                          constraints: BoxConstraints(maxWidth: 1200),
                          child: child
                      ),
                    )
                ),
                Container(
                  height: 20,
                  color: Colors.blue,
                  child: Text('footer', textAlign: TextAlign.center,),
                )
              ],
            )
        )
    );
  }
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
        visualDensity: VisualDensity.adaptivePlatformDensity,
      ),
      debugShowCheckedModeBanner: false,
      builder: (context, child) => TestLayout(child: child,),
      navigatorKey: GlobalKey<NavigatorState>(),
      onGenerateRoute: generateRoute,
      initialRoute: '/',
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key key, this.title}) : super(key: key);

  final String title;

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Test'),
      ),
      body: Container(
          width: 50,
          height: 50,
          child: FloatingActionButton(
            tooltip: 'Example',
            onPressed: (){},
          )
      ),
    );
  }
}
```


Also, flutter doctor output

[√] Flutter (Channel master, 1.20.0-8.0.pre.125, on Microsoft Windows [Version 10.0.17763.1217], locale uk-UA)
    • Flutter version 1.20.0-8.0.pre.125 at C:\src\flutter
    • Framework revision 9c4a5ef1ed (17 hours ago), 2020-07-15 21:41:27 +0100
    • Engine revision 774ae0313d
    • Dart version 2.9.0 (build 2.9.0-21.0.dev 89fe12b12a)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at C:\Users\aaa\AppData\Local\Android\sdk
    • Platform android-29, build-tools 29.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.6.2)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.6.30204.135
    • Windows 10 SDK version 10.0.18362.0

[√] Android Studio (version 3.6)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.8052
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)

[√] Connected device (3 available)
    • Windows (desktop) • windows    • windows-x64    • Microsoft Windows [Version 10.0.17763.1217]
    • Web Server (web)  • web-server • web-javascript • Flutter Tools
    • Chrome (web)      • chrome     • web-javascript • Google Chrome 84.0.4147.89

• No issues found!

**Comment by TahaTesser** [comment]
<details> 
<summary>flutter doctor -v</summary> 

```
[✓] Flutter (Channel dev, 1.20.0-7.1.pre, on Mac OS X 10.15.6 19G73, locale
    en-GB)
    • Flutter version 1.20.0-7.1.pre at /Users/taha/Code/flutter_dev
    • Framework revision 7736f3bc90 (6 days ago), 2020-07-10 16:33:05 -0700
    • Engine revision d48085141c
    • Dart version 2.9.0 (build 2.9.0-21.2.beta)

 
[✓] Android toolchain - develop for Android devices (Android SDK version
    30.0.0)
    • Android SDK at /Users/taha/Code/sdk
    • Platform android-30, build-tools 30.0.0
    • ANDROID_HOME = /Users/taha/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.6, Build version 11E708
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google
      Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.47.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.2

[✓] Connected device (5 available)
    • SM M305F (mobile)      • 32003c30dc19668f          • android-arm64  •
      Android 10 (API 29)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            •
      iOS 13.6
    • macOS (desktop)        • macos                     • darwin-x64     •
      Mac OS X 10.15.6 19G73
    • Web Server (web)       • web-server                • web-javascript •
      Flutter Tools
    • Chrome (web)           • chrome                    • web-javascript •
      Google Chrome 83.0.4103.116

• No issues found!
```

</details>

**Comment by romathebest** [comment]
@TahaTesser any news when it will be fixed? I have checked  #60479 and it fixes the problem but seems like it was closed
